apps/ui/components/calendar/ReservationCalendarControls.tsx

```diff
@@ -603,6 +605,9 @@ const ReservationCalendarControls = <T extends Record<string, unknown>>({
     (n) => n?.endDatetime
   );
 
+  let loc;
+  if (isBrowser) loc = window.location;
```

Comment:
not a fan of ternaries, but let on line 600 of a 2000 line function is worse.
`const loc = isBrowser ? window.location : undefined;`